src/Sustainsys.Saml2/Validation/AssertionValidator.cs

```diff
@@ -84,4 +91,50 @@ protected virtual void ValidateConditions(
         // to care about proxy restrictions. They are only valid if the SP is a proxy that acts as an IDP
         // to other applications. We'll just ignore proxy restrictions for now.
     }
+    /// <summary>
+    /// Validate Subject Confirmation of an assertion. 
+    /// </summary>
+    /// <param name="assertion"></param>
```

Comment:
Fill in XML docs descriptions of parameters. Can be one word each, but should at least be something.